examples/super-basic/index.html

```diff
@@ -2,7 +2,7 @@
   <title>Most Basic Super-Hands Example</title>
   <script src="https://aframe.io/releases/1.4.0/aframe.min.js"></script>
   <script src="https://cdn.jsdelivr.net/gh/donmccurdy/aframe-extras@v6.1.1/dist/aframe-extras.misc.min.js"></script>
-  <script src="https://rawgit.com/wmurphyrd/aframe-super-hands-component/master/dist/super-hands.js"></script>
+  <script src="https://rawgit.com/c-frame/aframe-super-hands-component/master/dist/super-hands.js"></script>
```

Comment:
rawgit service doesn't work anymore for new urls. You need to use jsdelivr.

Reply:
Thanks @vincentfretin - this page isn't linked anywhere and was just used internally to validate the basic example on the readme still worked. I'll make a ticket to address this later (maybe replace both with a glitch)